flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/api/functions/source/SourceFunction.java

```diff
@@ -36,9 +36,9 @@
 	/**
 	 * Main work method of the source. This function is invoked at the beginning of the

```

Comment:
Oversight on my part, and I forgot to rework the FileMonitoringFunction (a source)